### ISSUE TITLE
Additional examples needed in .md

### ISSUE THREAD

**Issue by gerbil** [peter-murray/node-teamcity-api#4]
Get project by locator for instance
- How to trigger a build?

**Comment by sashman** [comment]
Also use of templates

**Comment by peter-murray** [comment]
Agreed, the documentation is almost non-existent, as I was not finished defining the API itself when I wrote some of it.

I will sort this in the coming month, but for now if you want examples, the tests are great examples of using the API.

I don't think from memory I have added support for build triggering yet, but will prioritise that in the upcoming release.

**Comment by savimcio** [comment]
I think this issue can be closed now in documentation this information present as [Run a build configuration](https://github.com/peter-murray/node-teamcity-api#teamcitybuildsstartbuild---run-a-build-configuration). It was added in [this](https://github.com/peter-murray/node-teamcity-api/commit/e6866da23bf90e8a5a84398774d3cb85e8aba482#diff-04c6e90faac2675aa89e2176d2eec7d8) commit
